tls/s2n_connection.c

```diff
@@ -922,11 +922,9 @@ int s2n_connection_get_cipher_iana_value(struct s2n_connection *conn, uint8_t *f
     POSIX_ENSURE_MUT(second);
 
     /* ensure we've negotiated a cipher suite */
-    POSIX_ENSURE(memcmp(conn->secure->cipher_suite->iana_value,
-                         s2n_null_cipher_suite.iana_value, sizeof(s2n_null_cipher_suite.iana_value))
-                    != 0,
-            S2N_ERR_INVALID_STATE);
-
```

Comment:
nit: I think liked the extra newline here, since the comment above only corresponds to this comparison and not the following lines.